test.js

```diff
@@ -22,3 +23,29 @@ test('catch', async t => {
 
 	t.true(inRange(promise.time, 200, 250));
 });
+
+test.cb.serial('log', t => {
+	function myDelay(ms) {
+		return new Promise(resolve => {
+			setTimeout(resolve, ms);
+		});
+	}
```

Comment:
```js
const myDelay = ms => new Promise(resolve => {
	setTimeout(resolve, ms);
});
```

Per the ES2015 spec, JS engines are able to infer the function name even when assigned to a variable like here.